speech/google/cloud/speech/client.py

```diff
@@ -46,26 +46,17 @@ class Client(client_module.Client):
 
     _connection_class = Connection
 
-    def async_recognize(self, content, source_uri, encoding, sample_rate,
+    def async_recognize(self, encoding, sample_rate, content=None,
```

Comment:
Could be, I was just trying to be alphabetic. You can't have kwargs before args though right? Could make `encoding` and `sample_rate` defaulted to `None` as well though.